recipe/meta.yaml

```diff
@@ -98,7 +98,7 @@ outputs:
         - {{ pin_subpackage('libmamba', exact=True) }}
     test:
       requires:
-        - conda 4.12
+        - conda
```

Comment:
We could have removed conda entirely from the requires section as it was added to be specific in version. By default, it will be there even if we don't specify it here.